qemu/qmp.go

```diff
@@ -178,6 +178,36 @@ type MemoryDevices struct {
 	Type string            `json:"type"`
 }
 
+// CPUInstanceProperties represent cpu properties
+type CPUInstanceProperties struct {
```

Comment:
The structure seems to be identical to the existing CPUProperties structure.  Can we not just re-use this?  If so we'd probably need to update the GoDoc for that structure.

Reply:
You're right. I'll reuse it and remove the duplication.